src/pages/gen2/build-a-backend/auth/cognito-triggers/index.mdx

```diff
@@ -0,0 +1,162 @@
+export const meta = {
+  title: 'Cognito Triggers',
+  description: 'Learn how to create Cognito Triggers with Amplify (Gen 2) Auth'
```

Comment:
Learn how to create pre-sign up, or post-sign up handlers with Amplify (Gen 2) Auth